profiler/src/test/java/com/splunk/opentelemetry/profiler/AgentInternalsFilterTest.java

```diff
@@ -24,10 +24,10 @@
 
```

Comment:
This class seems to be missing some test cases:
* `includeAgentInternals = true`
* the two line case
* wrong arguments, i.e. string not starting with `"` or last index == -1
* start/end different from `0, stack.length()`